New Community Call

clcoffey opened the pull request:
Posting the new community call on the website and changing the banner on the home page.

lenucksi commented on board/2020-06-10-terse-notes.md:
```diff
@@ -0,0 +1,38 @@
+- Daniel Izquierdo
+- Russel Rutledge (Secretary)
+- Klaas-Jan Stol
+- Johannes Tigges
```
Legally I'm not a director, I'm also mentioned twice, also in line 34.

gruetter replied:
in time ;) ...

lenucksi commented on board/terse-notes-template.md:
```diff
@@ -0,0 +1,34 @@
+- Daniel Izquierdo
+- Russel Rutledge (Secretary)
+- Klaas-Jan Stol
+- Johannes Tigges
```
Probably want to fix that too.

clcoffey committed:
New Community Call

Posting the new community call on the website and changing the banner on the home page.

claredillon reviewed (approved):
Thanks Cristina -looks great
